docs/content/preview/yugabyte-voyager/release-notes.md

```diff
@@ -21,20 +21,21 @@ Voyager releases (starting with v2025.5.2) use the numbering format `YYYY.M.N`,
 
 ## New feature
 
-- Automatically apply performance optimizations (like removing redundant indexes) during export schema phase and export the optimized schema. A schema optimization report is also generated for user review.
+- Automatically apply performance optimizations recommended by the migration assessment (such as removing redundant indexes) during the export schema phase. Voyager also generates a schema optimization report detailing the optimizations that were applied. To turn on automatic optimization, set the new `--skip-performance-optimizations` flag in the `export schema` command to false.
```

Comment:
Oh sorry the default is on (skip is off - confusing!) - so

To turn automatic optimization off, set the new `--skip-performance-optimizations` flag in the `export schema` command to true.